.github/workflows/ci.yml

```diff
@@ -31,6 +36,3 @@ jobs:
           go mod tidy -v && git diff --no-patch --exit-code
           ./run-lint.sh
           go test -v -race ./...
-
-      - name: Build on ${{ matrix.goVer }}
-        run: go build ./...
```

Comment:
go test builds it already :)

Reply:
probably build was there, lint is added afterwards